src/main/java/org/openjdk/jextract/impl/HeaderFileBuilder.java

```diff
@@ -377,9 +377,9 @@ private void emitConstant(Class<?> javaType, String constantName, Object value,
 
     private String constantValue(Class<?> type, Object value) {
         if (value instanceof String) {
-            return STR."Arena.ofAuto().allocateFrom(\"\{Utils.quote(Objects.toString(value))}\");";
+            return STR."Arena.ofAuto().allocateFrom(\"\{Utils.quote(Objects.toString(value))}\")";
```

Comment:
The changes here are unrelated - but I noted a double `;` when eyeballing the generated sources for windows.h, and addressed it here